<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderItemsDialog</class>
 <widget class="QDialog" name="OrderItemsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListView" name="listView_items"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_buttons">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_up">
       <property name="text">
        <string>Up</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_down">
       <property name="text">
        <string>Down</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>OrderItemsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>OrderItemsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
